### DISCUSSION TITLE
Why does the some package ask for administrator permission

### DISCUSSION THREAD

**Discussion by 09bravo** [microsoft/winget-pkgs#152237] [Q&A] [answered]
Hello, using winget i tried to install "plover" (https://github.com/openstenoproject/plover) it asks for administrator permission is that normal ?, this is not the first time i have seen that so i do not know if that's ok or not, if it is then I think it should be hidden, anyway thanks.

**Comment by mdanish-kh** [comment, marked as answer]
Do you get the same while normally downloading and running the installer? At a high level, WinGet simply downloads the installer and runs it for you with appropriate silent switches. It does not do anything extra related to elevating permissions. If the installer requires administrator privileges, then winget won't bypass it for you.

You could try running winget in an administrative terminal to get rid of the prompt.
